### PR TITLE
add mir packages

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -51,7 +51,12 @@ USER d-user
 RUN cd /sandbox && for package_name in \
 		mir-algorithm \
 		mir-random \
+		mir-core \
+		mir-runtime \
+		mir-optim \
 		mir \
+		asdf \
+		stdx-allocator \
```
Comment (wilzbach) [inline Dockerfile:59]: Why this? The logic is smart enough to fetch dependencies.

**Review by wilzbach — COMMENTED** [review]
Maybe only add the dependencies that you actually need as the other ones are then automatically fetched and thus this might avoid some problems down the road.

File: Dockerfile (hunk 1 above)
Comment (9il) [reply Dockerfile:59]: A user may want to use a recent version a package, while a fetched dependency may not be recent.

Comment (PetarKirov) [reply Dockerfile:59]: Fair enough.

**Review by PetarKirov — APPROVED** [review]

**Pull request merged**